stdlib/concurrent/futures/_base.pyi

```diff
@@ -76,12 +76,12 @@ def as_completed(fs: Iterable[Future[_T]], timeout: float | None = ...) -> Itera
 
 # Ideally this would be a namedtuple, but mypy doesn't support generic tuple types. See #1976
 class DoneAndNotDoneFutures(Sequence[Set[Future[_T]]]):
-    done: Set[Future[_T]]
-    not_done: Set[Future[_T]]
-    def __new__(_cls, done: Set[Future[_T]], not_done: Set[Future[_T]]) -> DoneAndNotDoneFutures[_T]: ...
+    done: set[Future[_T]]
+    not_done: set[Future[_T]]
+    def __new__(_cls, done: set[Future[_T]], not_done: set[Future[_T]]) -> DoneAndNotDoneFutures[_T]: ...
```

Comment:
Should `done` and `not_done` be mutable, so that users can add or remove futures from them? `collections.abc.Set` is immutable and would prevent that.

Reply:
They are `set` in the implementation and mutable.